lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/entry/InformationEntry.java

```diff
@@ -25,6 +25,12 @@
      */
     public static final String ACTION_DISPLAY_APPROVE_MESSAGE = "com.pax.us.pay.action.DISPLAY_APPROVE_MESSAGE";
 
-
+    /**
+     * The intent action of display transaction information <br>
+     * input: {@link EntryExtraData#PARAM_PACKAGE} <br>
+     * input: {@link EntryExtraData#PARAM_MESSAGE} <br>
+     * request: None <br>
+     */
+    public static final String ACTION_DISPLAY_TRANS_END = "com.pax.us.pay.action.DISPLAY_TRANS_END";//added for visa installment
```

Comment:
1. Is it a page action or a status?
2. Is the action name correct, display trans end, transaction information? does it relate to visa installment?

Reply:
This is a page action. Visa wants us to display a certain page when a visa installment transaction is approved. I think the name is not aacurate enough, I will modify them during the next release which will happen in this week.

Reply:
Got it.
And please modify the descriptions as well.

Reply:
@MichealUfl I didn't find update on any commits, Please modify it ASAP.

Reply:
See df474c0